ln-simln-jamming/src/main.rs

```diff
@@ -276,25 +255,45 @@ async fn main() -> Result<(), BoxError> {
         })
         .collect();
 
+    // Before running the simulation, run any setup needed for the attack.
+    let attack_setup = attack.setup_for_attack(&attacker_nodes).await?;
+    for (channel, pubkey) in attack_setup.general_jammed_nodes.iter() {
+        reputation_interceptor
+            .lock()
+            .await
+            .jam_channel(pubkey, *channel)
+            .await?;
+    }
+
     let attacker_actions_shutdown = shutdown.clone();
+    let attack_custom_actions = Arc::clone(&attack);
+    let custom_actions_listener = listener.clone();
     tokio::spawn(async move {
         if let Err(e) = attack_custom_actions
-            .run_custom_actions(attacker_nodes, listener.clone())
+            .run_custom_actions(attacker_nodes, custom_actions_listener)
             .await
         {
             log::error!("Error running custom attacker actions: {e}");
             attacker_actions_shutdown.trigger();
         }
     });
 
+    // Spawn a task that will trigger shutdown of the simulation if the attacker loses reputation provided that the
```

Comment:
While we're here update this comment? Now that we've moved the reputation check into the sink attack specifically.